app/lib/constants/role.rb

```diff
@@ -16,6 +16,7 @@ module Role
                      "Resource Admin: Page",
                      "Resource Admin: FeedbackMessage",
                      "Resource Admin: Config",
-                     "Resource Admin: Broadcast"].freeze
+                     "Resource Admin: Broadcast",
+                     "Resource Admin: DisplayAd"].freeze
```

Comment:
I'm not 100% sure this had to be added, I think it's helpful but open to take it off if anyone feels differently :)

Reply:
I'm curious about whether this needs to be added or not, too! 😄